src/pages/RFOX/components/Stake/Bridge/BridgeConfirm.tsx

```diff
@@ -127,15 +127,23 @@ export const BridgeConfirm: FC<BridgeRouteProps & BridgeConfirmProps> = ({ confi
           <Stack alignItems='center'>
             <AssetIcon size='sm' assetId={sellAsset?.assetId} />
             <Stack textAlign='center' spacing={0}>
-              <Amount.Crypto value={bridgeAmountCryptoPrecision} symbol={sellAsset.symbol} />
+              <Amount.Crypto
+                prefix='≈'
```

Comment:
💜

Reply:
Fixed in [91d7519](https://github.com/shapeshift/web/pull/7241/commits/91d75198190530c4b54f25776c78f4efd02293ba)